doc/user_manual/sampler.tex

```diff
@@ -105,6 +105,14 @@ \section{Samplers}
   \end{itemize}
 }
 
+\newcommand{\constantVariablesDescription}
+{
+  \xmlNode{constant}, \xmlDesc{XML node,
+  optional parameter} the user is able to input variable that needs to be kept constant.    
```

Comment:
Probably should be "able to input variables that need to be" or "able to input a variable that needs to be" or even "able to add an input variable that needs to be".